pyamg/relaxation/smoothing.py

```diff
@@ -123,6 +123,8 @@ def change_smoothers(ml, presmoother, postsmoother):
     >>> x = ml.solve(b, tol=1e-8, residuals=residuals)
     '''
 
+    ml.symmetric_smoothing = True
```

Comment:
The return values for this function in the function documentation should also list that you set ml.symmetric_smoothing